.changes/next-release/bugfix-TraceId-22972.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
We should remove this changelog entry. This change nor rollback never made it to v2.